userguide/conf.py

```diff
@@ -85,6 +85,8 @@
 .. |ctrlr-term-1|         replace:: TrueNAS controller 1
 .. |ctrlr-term-2|         replace:: TrueNAS controller 2
 .. |ctrlr-term-1-2|       replace:: TrueNAS controller 1/2
+.. |ctrlr-term-primary|   replace:: active TrueNAS controller
```

Comment:
Why not name these for what the correct (currently) terms are, |ctrlr-term-active| and |ctrlr-term-standby|?

Reply:
I was trying to make a general descriptive term that would be applicable regardless of the currently used term.